int32 route_index
float64 lateral_offset
bool end_of_path
